flox-bash/tests/package.bats

```diff
@@ -7,46 +7,62 @@ load test_support.bash
 
 @test "flox package sanity check" {
   # directories
-  [ -d $FLOX_PACKAGE/bin ]
-  [ -d $FLOX_PACKAGE/libexec ]
-  [ -d $FLOX_PACKAGE/libexec/flox ]
-  [ -d $FLOX_PACKAGE/etc ]
-  [ -d $FLOX_PACKAGE/etc/flox.zdotdir ]
-  [ -d $FLOX_PACKAGE/lib ]
-  [ -d $FLOX_PACKAGE/share ]
-  [ -d $FLOX_PACKAGE/share/man ]
-  [ -d $FLOX_PACKAGE/share/man/man1 ]
-  [ -d $FLOX_PACKAGE/share/bash-completion ]
-  [ -d $FLOX_PACKAGE/share/bash-completion/completions ]
+  [ -d "$FLOX_PACKAGE/bin" ]
+  [ -d "$FLOX_PACKAGE/libexec" ]
+  [ -d "$FLOX_PACKAGE/libexec/flox" ]
+  [ -d "$FLOX_PACKAGE/etc" ]
+  [ -d "$FLOX_PACKAGE/etc/flox.zdotdir" ]
+  [ -d "$FLOX_PACKAGE/lib" ]
+  [ -d "$FLOX_PACKAGE/share" ]
+  [ -d "$FLOX_PACKAGE/share/man" ]
+  [ -d "$FLOX_PACKAGE/share/man/man1" ]
+  [ -d "$FLOX_PACKAGE/share/bash-completion" ]
+  [ -d "$FLOX_PACKAGE/share/bash-completion/completions" ]
   # executables
-  [ -x $FLOX_CLI ]
-  [ -x $FLOX_PACKAGE/libexec/flox/gh ]
-  [ -x $FLOX_PACKAGE/libexec/flox/nix ]
-  [ -x $FLOX_PACKAGE/libexec/flox/flox ]
+  [ -x "$FLOX_CLI" ]
+  [ -x "$FLOX_PACKAGE/libexec/flox/gh" ]
+  [ -x "$FLOX_PACKAGE/libexec/flox/nix" ]
+  [ -x "$FLOX_PACKAGE/libexec/flox/flox" ]
   # Could go on ...
 }
 
 @test "flox --prefix" {
-  run $FLOX_CLI --prefix
+  run "$FLOX_CLI" --prefix
   assert_success
-  assert_output $FLOX_PACKAGE
+  assert_output "$FLOX_PACKAGE"
 }
 
 @test "flox install by /nix/store path" {
-  run $FLOX_CLI install -e $TEST_ENVIRONMENT $FLOX_PACKAGE
+  run "$FLOX_CLI" install -e "$TEST_ENVIRONMENT" "$FLOX_PACKAGE"
   assert_success
   assert_output --partial "Installed '$FLOX_PACKAGE' package(s) into '$TEST_ENVIRONMENT' environment."
 }
 
 @test "flox list after installing by store path should contain package" {
-  run $FLOX_CLI list -e $TEST_ENVIRONMENT
+  run "$FLOX_CLI" list -e "$TEST_ENVIRONMENT"
   assert_success
   assert_output --partial "Curr Gen  1"
   assert_output --partial "0  $FLOX_PACKAGE  $FLOX_PACKAGE_FIRST8"
 }
 
+@test "flox activate on multiple environments" {
```

Comment:
I think this belongs in integration.bats. Tests in package.bats depend on `FLOX_PACKAGE`. They were split out because of the flox and flox-bash split

Reply:
moved it.